x-pack/plugins/enterprise_search/public/applications/shared/role_mapping/role_mappings_table.test.tsx

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { shallow } from 'enzyme';
+
+import { EuiFieldSearch, EuiTableRow, EuiTextColor } from '@elastic/eui';
+
+import { wsRoleMapping, asRoleMapping } from './__mocks__/roles';
+
+import { ALL_LABEL } from './constants';
+
+import { RoleMappingsTable } from './role_mappings_table';
+
+describe('RoleMappingsTable', () => {
+  const getRoleMappingPath = jest.fn();
+  const roleMappings = [wsRoleMapping] as any;
```

Comment:
```suggestion
const roleMappings = [
    {
      ...wsRoleMapping,
      accessItems: [
        {
          name: 'foo',
        },
      ],
    },
  ];
```

Again, I think it's best to avoid `any`s. Especially here... this data is actually missing a required prop `RoleMappingTable`, which is the `accessItems` prop on each role mapping.